app/services/decorators/v5/result_summary_decorator.rb

```diff
@@ -26,7 +27,8 @@ def as_json
                                                                  gross_income_summary,
                                                                  partner_present: assessment.partner.present?).as_json,
           partner_disposable_income:,
-          capital: CapitalResultDecorator.new(capital_summary).as_json,
+          capital: CapitalResultDecorator.new(capital_summary,
+                                              @calculation_output&.capital_subtotals&.applicant_capital_subtotals).as_json,
```

Comment:
are all these safe navigation operators necessary?

Reply:
Not any more! Removing them.